### ISSUE TITLE
checked-c-convert tool: struct variable containing a checked pointer not initialized

### ISSUE THREAD

**Issue by hasantouma** [microsoft/checkedc-clang#621]
I noticed this issue when converting the [Icecast](http://icecast.org/) code. In the file `src/thread/avl.c` and various others.

The C compiler doesn't complain when a struct variable is not initialized, but the Checked C compiler does. The `checked-c-convert` tool doesn't give an warnings that an initialized struct variable was seen.

I created a small example for reference:

Original C code:
```
typedef struct _link_node {
  struct _link_node *parent;
  int width;
} link_node;

int add_to_link_node(link_node *node) {

  node->width += 5;

  return node->width;
}

int main() {

  link_node here;
  here.width = 10;

  return add_to_link_node(&here);
}
```

`checked-c-convert` output:
```
typedef struct _link_node {
  _Ptr<struct _link_node> parent;
  int width;
} link_node;

int add_to_link_node(_Ptr<link_node> node) {

  node->width += 5;

  return node->width;
}

int main() {

  link_node here;
  here.width = 10;

  return add_to_link_node(&here);
}
```

The `checked-c-convert` output will produce the error:
`error: struct variable 'here' containing a checked pointer must have an initializer`

**Comment by Machiry** [comment]
This is a duplicate of the issue: https://github.com/plum-umd/checkedc-clang/issues/3 which we fixed in the plum repo: https://github.com/plum-umd/checkedc-clang/commit/6bdcacdc6dfdc0fc9c5a8301fc38fed08114ceae and raised a pull-request (https://github.com/microsoft/checkedc-clang/pull/679) for the same.

**Comment by mgrang** [comment]
> This is a duplicate of the issue: [plum-umd#3](https://github.com/plum-umd/checkedc-clang/issues/3) which we fixed in the plum repo: [plum-umd@6bdcacd](https://github.com/plum-umd/checkedc-clang/commit/6bdcacdc6dfdc0fc9c5a8301fc38fed08114ceae) and raised a pull-request (#679) for the same.

@Machiry Could you please go ahead and close this issue.

**Comment by Machiry** [comment]
I cannot close this. I am not the owner of this issue or the project. Could you close this, please!
